AzureCalling/Controllers/HangupConfirmationViewController.swift

```diff
@@ -19,12 +19,16 @@ class HangupConfirmationViewController: UIViewController {
         view.backgroundColor = UIColor(red: 0, green: 0, blue: 0, alpha: 0.8)
         view.isOpaque = false
 
+        let primaryColor = UIColor(named: "PrimaryColor") ?? .systemBlue
```

Comment:
Our color lookups should never be null. So we shouldn't need this conditional.
Could we define some app colors constants instead?

Reply:
Add ColorTheme as a helper class
UIColor(named:) could return nil, provide a fallback value avoid crash